policy-controller/grpc/src/lib.rs

```diff
@@ -696,20 +678,28 @@ fn convert_outbound_http_route(
     let rules = rules
         .into_iter()
         .map(|OutboundHttpRouteRule { matches, backends }| {
-            let mut backends = backends
+            let backends = backends
                 .into_iter()
                 .map(convert_http_backend)
                 .collect::<Vec<_>>();
-            if backends.is_empty() {
-                backends = vec![default_backend.clone()];
-            }
-            outbound::http_route::Rule {
-                matches: matches.into_iter().map(http_route::convert_match).collect(),
-                backends: Some(outbound::http_route::Distribution {
+            let dist = if backends.is_empty() {
+                outbound::http_route::Distribution {
+                    kind: Some(outbound::http_route::distribution::Kind::FirstAvailable(
+                        outbound::http_route::distribution::FirstAvailable {
+                            backends: vec![default_backend.clone()],
+                        },
+                    )),
+                }
+            } else {
+                outbound::http_route::Distribution {
                     kind: Some(outbound::http_route::distribution::Kind::RandomAvailable(
                         outbound::http_route::distribution::RandomAvailable { backends },
                     )),
-                }),
+                }
+            };
+            outbound::http_route::Rule {
+                matches: matches.into_iter().map(http_route::convert_match).collect(),
+                backends: Some(dist),
                 filters: Default::default(),
             }
```

Comment:
nit, TIOLI: we could make this a couple lines shorter by just having the `if` evaluate to the `distribution::Kind` and wrapping it in the outer struct once:
```rust

            let dist = if backends.is_empty() {
                outbound::http_route::distribution::Kind::FirstAvailable(
                    outbound::http_route::distribution::FirstAvailable {
                        backends: vec![default_backend.clone()],
                    },
                )
            } else {
                outbound::http_route::distribution::Kind::RandomAvailable(
                    outbound::http_route::distribution::RandomAvailable { backends },
                ),
            };
            outbound::http_route::Rule {
                matches: matches.into_iter().map(http_route::convert_match).collect(),
                backends: Some(outbound::http_route::Distribution { kind: Some(dist) }),
                filters: Default::default(),
            }
```
(github won't let me leave this as a suggestion(
```